torchrec/csrc/dynamic_embedding/details/bits_op.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <cstdint>
+#include <type_traits>
+
+namespace torchrec::dynamic_embedding::details {
+
+namespace bits_impl {
+template <typename T>
+struct Clz {
+  int operator()(T v) const;
+};
+
+template <typename T>
+struct Ctz {
+  int operator()(T v) const;
+};
+} // namespace bits_impl
+
+/**
```

Comment:
also explain that ctz means 'count trailing zeros' and clz means 'count leading zeros'

Reply:
added.